x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/shared/map_processor_type_to_form.tsx

```diff
@@ -246,210 +371,373 @@ export const mapProcessorTypeToDescriptor: MapProcessorTypeToDescriptor = {
     docLinkPath: '/split-processor.html',
     label: i18n.translate('xpack.ingestPipelines.processors.label.split', {
       defaultMessage: 'Split',
     }),
-    description: i18n.translate('xpack.ingestPipelines.processors.description.split', {
+    typeDescription: i18n.translate('xpack.ingestPipelines.processors.description.split', {
       defaultMessage: 'Splits a field value into an array.',
     }),
+    getDefaultDescription: ({ field, target_field: targetField = field }) =>
+      i18n.translate('xpack.ingestPipelines.processors.defaultDescription.split', {
+        defaultMessage: 'Splits the string stored in "{field}" to an array in "{target_field}"',
+        values: {
+          field,
+          target_field: targetField,
+        },
+      }),
   },
   trim: {
     FieldsComponent: Trim,
     docLinkPath: '/trim-processor.html',
     label: i18n.translate('xpack.ingestPipelines.processors.label.trim', {
       defaultMessage: 'Trim',
     }),
-    description: i18n.translate('xpack.ingestPipelines.processors.description.trim', {
+    typeDescription: i18n.translate('xpack.ingestPipelines.processors.description.trim', {
       defaultMessage: 'Removes leading and trailing whitespace from a string.',
     }),
+    getDefaultDescription: ({ field }) =>
+      i18n.translate('xpack.ingestPipelines.processors.defaultDescription.trim', {
+        defaultMessage: 'Trims whitespaces from "{field}"',
+        values: {
+          field,
+        },
+      }),
   },
   uppercase: {
     FieldsComponent: Uppercase,
     docLinkPath: '/uppercase-processor.html',
     label: i18n.translate('xpack.ingestPipelines.processors.label.uppercase', {
       defaultMessage: 'Uppercase',
     }),
-    description: i18n.translate('xpack.ingestPipelines.processors.description.uppercase', {
+    typeDescription: i18n.translate('xpack.ingestPipelines.processors.description.uppercase', {
       defaultMessage: 'Converts a string to uppercase.',
     }),
+    getDefaultDescription: ({ field }) =>
+      i18n.translate('xpack.ingestPipelines.processors.defaultDescription.uppercase', {
+        defaultMessage: 'Converts value stored in "{field}" to uppercase',
+        values: {
+          field,
+        },
+      }),
   },
   urldecode: {
     FieldsComponent: UrlDecode,
     docLinkPath: '/urldecode-processor.html',
     label: i18n.translate('xpack.ingestPipelines.processors.label.urldecode', {
       defaultMessage: 'URL decode',
     }),
-    description: i18n.translate('xpack.ingestPipelines.processors.description.urldecode', {
+    typeDescription: i18n.translate('xpack.ingestPipelines.processors.description.urldecode', {
       defaultMessage: 'Decodes a URL-encoded string.',
     }),
+    getDefaultDescription: ({ field, target_field: targetField = field }) =>
+      i18n.translate('xpack.ingestPipelines.processors.defaultDescription.url_decode', {
+        defaultMessage: 'Decodes the URL string in "{field}" to "{target_field}"',
```

Comment:
Specifying `"{target_field}"` doesn't include it in the default message:
![image](https://user-images.githubusercontent.com/25848033/113345482-84123180-9300-11eb-8704-029e24e53b8b.png)